Sources/UIImageExtensions.swift

```diff
@@ -9,6 +9,12 @@
 import UIKit
 
 extension UIImage {
+    
+    /// EZSE: Returns base64 string
+    var base64: String {
+        return UIImageJPEGRepresentation(self, 1.0)!.base64EncodedString()
```

Comment:
Yes I was able to get it to work. My prototype code was : 

    public func testImageBase64() {
        let image = UIImage(contentsOfFile: "/workplace/arunavs/third_party/EZSwiftExtensions/charizard.png")
        print(image?.base64)
    }

It gave a non nil base 64 encoded string. 

Can you write a test(with the path formatted as a relative path) to assert that the output is not nil and also its of base 64 format (do we have an extension/foundation API for that?). I will approve this after that.